mathics/core/characters.py

```diff
@@ -30,1784 +31,2232 @@
 \uF82A\uF82B\uF82C\uF82D\uF82E\uF82F\uF830\uF831\uF832\uF833\uFE35\uFE36\
 \uFE37\uFE38"
 
-# All supported longname characters
```

Comment:
`named_characters` appears in `mathics/core/parser/prescanner.py` and it looks like here that name is removed. Does it appear somewhere else? -  I can't find it in this PR.

Reply:
`named_characters` was in there, but thare was so much junk around it no wonder you couldn't find it. I've extracted all of the data to a separate YAML file to improve readability (it's now easy to find where `named_characters` is).

Reply:
See https://github.com/mathics/Mathics/pull/1109/commits/552eea6f1018bdb9f4110c3783a2b2036cd16f27